app/templates/frameworks/_framework_actions.html

```diff
@@ -14,8 +14,9 @@
   {% endif %}
 
   {% if supplier_is_on_framework and not supplier_framework.agreementReturned %}
+  {% set sign_agreement_route = url_for('.legal_authority', framework_slug=framework.slug) if is_e_signature_supported_framework else url_for('.framework_agreement', framework_slug=framework.slug) %}
```

Comment:
I've spotted two more includes that the old URL is used - https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/templates/suppliers/_frameworks_standstill.html#L32 and https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/templates/suppliers/_frameworks_live.html#L6

Reply:
good spot @katstevens 🕵️‍♀️  .
I've added the same logic to these 2 templates now in 4dbb001.
As the framework wasn't passed directly in from the view I couldn't figure out a way to implement this except directly in the templates. Let me know if you have strong feelings about this and if so any way you can think to abstract it out.

Reply:
Using the config will work OK. However the `frameworks` list being passed to these templates is in fact a list of 'enhanced' frameworks annotated with extra info. We can add in a `is_e_signature_supported_framework` key in the `suppliers.dashboard` view here: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/main/views/suppliers.py#L83

Reply:
thanks for the tip, I have done this refactor in 9582ab7.

Reply:
Great - looks good to me.